components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -272,18 +275,8 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         break;
     }
 
-    return <StartPage phase={phase} error={!!error} title={title}>
+    return <StartPage phase={phase} error={error} title={title}>
       {statusMessage}
-      {error && <div>
-        <button className="mt-8 secondary" onClick={() => this.redirectTo(gitpodHostUrl.asDashboard().toString())}>Go back to dashboard</button>
-        <p className="mt-14 text-base text-gray-400 flex space-x-2">
-          <a href="https://www.gitpod.io/docs/">Docs</a>
-          <span>—</span>
-          <a href="https://status.gitpod.io/">Status</a>
-          <span>—</span>
-          <a href="https://www.gitpod.io/blog/">Blog</a>
-        </p>
-      </div>}
```

Comment:
**praise**: Nice removal! We could also remove these on Limits Reached modals but feel free to leave this as is for now.

Reply:
Done 🙌